analyzers/src/SonarAnalyzer.Common/Extensions/CountComparisonResultExtensions.cs

```diff
@@ -73,7 +76,7 @@ comparison switch
                 CountComparisonResult.Empty,
                 CountComparisonResult.AlwaysFalse);
 
-        private static CountComparisonResult Check(int count, int limit, CountComparisonResult smallerThanLimit, CountComparisonResult sameAsLimit, CountComparisonResult higherThanLimit)
+        private static CountComparisonResult Check(int count, int limit, CountComparisonResult higherThanLimit, CountComparisonResult sameAsLimit, CountComparisonResult smallerThanLimit)
```

Comment:
What happened here? How did we passed UTs if this was swapped?

I'd prefer `small, same, high` as a logical order.

Reply:
> What happened here? How did we passed UTs if this was swapped?

For Any/Empty we're only interested in equal to the limit. That's why. I was shocked to find this out.

> I'd prefer `small, same, high` as a logical order.

I first wanted to make things work, as I expected some discussion about this. :)

Reply:
After writing a test, I could safely refactor. I think it is better now.